debian/tests/run-tests.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -exuo pipefail
+
+export AUTHD_SKIP_EXTERNAL_DEPENDENT_TESTS=1
+export GOPROXY=off
+export GOTOOLCHAIN=local
+
+go test -mod=vendor -v ./...
```

Comment:
Using `-mod=vendor` is not necessary (and it only creates confusion, tbh): "By default, if a vendor directory is present and the go version in go.mod is 1.14 or higher, the go command acts as if -mod=vendor were set."

Reply:
oh, sorry I thought I had done... let me add a fixup